CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Projectiles/Fireball/FireballIdle.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System.Collections.Generic;
+
+namespace CrossPlatformDesktopProject.NPC
+{
+    class FireballIdle : INpcState
```

Comment:
Similar to the BoomerangIdle class - could be better to implement a method to make it go away rather than have an empty class and setting the boomerang object to it.

Reply:
Previous comment also applies